HealthcareSystem/PatientWebAppUnitTests/CreateObjectsForTests/TestObjectFactory.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace PatientWebAppTests.CreateObjectsForTests
+{
+    public class TestObjectFactory
+    {
+        public ICreateTestObject GetObject(string objectType)
```

Comment:
Obviously, if you decide to remove this interface this class goes as well, but I have two suggestions on what to do with this class if the interface is kept and made generic:
1. Keep this class, but split this method into three. This makes sense if you expect to have multiple implementations of ICreateTestObject<PatientDTO>, for example, or if you really just want your tests to not instantiate the creator objects directly.
2. Get rid of this class entirely anyway, and just instantiate the correct creator in your tests directly.